### PR TITLE
Torchscript export

### PR DESCRIPTION
**Pull request by mwisniowski**
- Training script now exports model as torchscript
- Changed YOLO_CHECKPOINT_PATH to YOLO_SAVE_DIR
  - Change default checkpoint dir to /yolov5/
  - Required to be able to point tensorboard config to this directory

### CODE REVIEW

File: export.py [hunk 1]
```diff
@@ -609,7 +609,7 @@ def run(
     return f  # return list of exported files/dirs
 
 
-def parse_opt():
+def parse_opt(known=False):
```
Comment (apstuff) [inline export.py:612]: Why did you need to change this?

Comment (mwisniowski) [reply export.py:612]: I want the default args of `export.py` and overwrite them with my own arguments. Probably the same reason you changed this in`val.py`.
